### ISSUE TITLE
Minor format change on aireplay-ng man page

### ISSUE THREAD

**Issue by aircrack-ng** [aircrack-ng/aircrack-ng#144]
**Reported by latinsud on 18 Mar 2007 12:43 UTC**

Due to a user not understanding the man page of aireplay-ng, he didn't know that -1 requires an argument. I've been watching other apps man pages and i suggest:

Before:
```
-1, --fakeauth=<delay>
              Fake authentication with AP.
```

After:
```
-1 delay, --fakeauth=<delay>
              Fake authentication with AP.
```

or:
```
-1 delay
--fakeauth=<delay>
              Fake authentication with AP.
```

**Comment by aircrack-ng** [comment]
**Comment by misterx on 18 Mar 2007 12:43 UTC**

(In #241) Avoid misunderstanding for 2 attacks that needs arguments (Closes: #144)

**Comment by aircrack-ng** [comment]
**Modified by misterx on 18 Mar 2007 12:43 UTC**
